package-lock.json

```diff
@@ -68,6 +68,7 @@
         "lodash": "4.17.21",
         "lottie-react-native": "6.5.1",
         "mapbox-gl": "^2.15.0",
+        "moize": "^6.1.6",
```

Comment:
do we need to commit this package-lock change because it's a new module? Or should we not push this file. Not sure how it works

Reply:
@dangrous I am not sure what is the practice in Expensify. Generally it's been recommended to commit .lock file as it is required on CI (with `npm ci`), but repo is relying on overrides as well so it may differ.

Reply:
I'd say if the `.lock` file is already included in the version control, then changes to it should also be reflected there. 

Take it with a grain of salt as I am not aware of all the practices in the repo. If you could ask someone you know possesses the knowledge, thanks!

Reply:
Ah I found in our docs, this should be good! I don't know why I thought otherwise.

Reply:
Thanks for checking that!